src/components/Home.tsx

```diff
@@ -0,0 +1,119 @@
+import { useContext, useEffect, useState } from 'react'
+import DisplayPatientResult from './DisplayPatientResult'
+import Pagination from './Pagination'
+import { AppContext } from '../context/AppContext'
+import Header from './Header'
+import swal from 'sweetalert'
+import ClipLoader from "react-spinners/ClipLoader";
+import { useNavigate } from 'react-router-dom'
+import { IoFilterSharp } from 'react-icons/io5'
+ 
+
+interface Result {
+    newData: []
+}
+const Home = () => {
+    const [patientInfo, setPatientInfo] = useState<any>([])
+    const [currentPage, setCurrentPage] = useState<number>(1)
+    const [patientsPerPage] = useState<number>(5)
+    const [searchParams, setSearchParams] = useState<string>('')
+    const [Loading,isLoading] = useState(false)
+    const [isTrue, setIsTrue] = useState<boolean>(false)
+
+    const { searchPatient, patients } = useContext(AppContext)
+
+    const navigate = useNavigate()
+
+    
+
+    const indexOfLastPatient = currentPage * patientsPerPage;
+    const indexOfFirstPatients = indexOfLastPatient - patientsPerPage;
+    const currentPatients = patients.slice(indexOfFirstPatients, indexOfLastPatient)
+
+    const paginate = (pageNumber: number) => setCurrentPage(pageNumber)
+
+    useEffect(()=>{
+        const banner = localStorage.getItem("Banner")
+          if(banner!==null){
```

Comment:
Why not equate to the actual value? I think the authentication component should handle the logic to show a successful toast, and you don't need local storage for this.

Reply:
:+1: